src/main/frontend/src/pages/Account.js

```diff
@@ -1,8 +1,17 @@
+import { useContext } from "react";
+import { AuthContext } from "../components/loginAndRegistration/AuthContext.js";
 import Profile from "../components/loginAndRegistration/Profile.js"
 
 function Account(){
+    const { loggedIn } = useContext(AuthContext);
     return (
-        <Profile/>
+        <div>
+            {loggedIn ? (
+                <Profile/>
+            ) : (
+                <p>You are not logged in!</p>
+            )}
```

Comment:
You can avoid `()` in ternary operator:
 `{ loggedIn ? <Profile/> : <p>You are not logged in!</p> }`